### ISSUE TITLE
Warnings in Point_set_shape_detection_3

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#2110]
The column "Debian-Testing" of recent testsuite show two warnings in point set shape detection and related line:
```
[...]/include/CGAL/Shape_detection_3/Cone.h:428:12: warning: variable 'radPerDist' set but not used [-Wunused-but-set-variable]
[...]/include/CGAL/Shape_detection_3/Octree.h:666:14: warning: variable 'v1' set but not used [-Wunused-but-set-variable]
```
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.11-Ic-12.shtml#Point_set_3_Examples

I saw the warnings in CGAL-4.11-Ic-12, but I think the bugs are in `master`.

If you see that the bug in in `releases/CGAL-4.10-branch` too, please create the PR on top of it.

**Comment by lrineau** [comment]
I confirm for 4.10-beta:
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.10-I-192.shtml#Point_set_3_Examples

**Comment by lrineau** [comment]
Fixed by #2114.
